### PR TITLE
BAEL-1466 Added code sample for apache-geode

### PR DESCRIPTION
**Pull request by sandy03934**

### CODE REVIEW

File: apache-geode/src/main/java/com/baeldung/geode/Customer.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public Customer(CustomerKey key, String firstName, String lastName, int age) {
+        this(firstName, lastName, age);
+        this.key = key;
+    }
+
+    // setters and getters
+
+    public static long getSerialVersionUID() {
```
Comment (jzheaux) [inline apache-geode/src/main/java/com/baeldung/geode/Customer.java]: Does Geode need this? Otherwise, let's take it out.

File: apache-geode/src/main/java/com/baeldung/geode/functions/PrimeNumber.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import org.apache.geode.cache.execute.Function;
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.cache.execute.RegionFunctionContext;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class PrimeNumber implements Function {
```
Comment (jzheaux) [inline apache-geode/src/main/java/com/baeldung/geode/functions/PrimeNumber.java]: Is this file still necessary? If not, let's remove it.

File: apache-geode/src/main/java/com/baeldung/geode/functions/CustomerWithMaxAge.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.execute.Function;
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.cache.execute.RegionFunctionContext;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Optional;
+
+public class CustomerWithMaxAge implements Function<Customer> {
```
Comment (jzheaux) [inline apache-geode/src/main/java/com/baeldung/geode/functions/CustomerWithMaxAge.java]: Is this class still necessary? If not, let's remove it.

File: apache-geode/src/main/java/com/baeldung/geode/Customer.java (hunk 1 above)
Comment (sandy03934) [reply apache-geode/src/main/java/com/baeldung/geode/Customer.java]: I was getting runtime issues which were solved by implementing serialversionUID

File: apache-geode/src/main/java/com/baeldung/geode/functions/PrimeNumber.java (hunk 2 above)
Comment (sandy03934) [reply apache-geode/src/main/java/com/baeldung/geode/functions/PrimeNumber.java]: Ok Sure..

File: apache-geode/src/main/java/com/baeldung/geode/functions/CustomerWithMaxAge.java (hunk 3 above)
Comment (sandy03934) [reply apache-geode/src/main/java/com/baeldung/geode/functions/CustomerWithMaxAge.java]: Ok Sure ...
